src/components/Common/UI/ProgressBreadcrumbs/ProgressBreadcrumbs.tsx

```diff
@@ -0,0 +1,80 @@
+import classnames from 'classnames'
+import type { CustomTypeOptions } from 'i18next'
+import { useTranslation } from 'react-i18next'
+import { Flex } from '../../Flex'
+import type { Breadcrumb, ProgressBreadcrumbsProps } from './ProgressBreadcrumbsTypes'
+import styles from './ProgressBreadcrumbs.module.scss'
+import { componentEvents } from '@/shared/constants'
+
+export function ProgressBreadcrumbs({
```

Comment:
Nit: I'd recommend just `Breadcrumbs` to better align with other design system naming